### PR TITLE
CODEOWNERS, add @Azure/api-stewardship-board to data-plane

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
Add github team `@Azure/api-stewardship-board` to codeowner as reviewer for data-plane PR.

# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
@mikekistler 

Let me know if the change is OK for you.

Currently there is no reviewer on purview data-plane, which causes PR without reviewers e.g. https://github.com/Azure/azure-rest-api-specs/pull/22241

I've also added `@Azure/api-stewardship-board` to the 2 other RPs that only have you as reviewer. Let me know if you'd prefer I don't update for these RPs.

File: CODEOWNERS [hunk 1]
```diff
@@ -173,7 +173,7 @@
 /specification/powerbidedicated/ @tarostok
 
 # PRLabel: %Purview
-/specification/purview/data-plane
+/specification/purview/data-plane @Azure/api-stewardship-board
```
Comment (mikekistler) [inline CODEOWNERS:176]:
```suggestion
/specification/purview/data-plane @mikekistler @Azure/api-stewardship-board
```

**Review by mikekistler — APPROVED** [review]
Looks good. 👍 

I added myself to Purview because I am the SDK PM for Purview and would like to be tagged on all their API reviews.

**Pull request merged**